ui/src/util/dates.js

```diff
@@ -9,6 +9,14 @@ export const convertToStandardDate = function (
   return moment(value, givenFormat).format("DD MMM YYYY");
 };
 
+export const isTodayAfterGivenDate = function (
+  givenDate,
+  givenDateFormat = pythonDefaultDateFormat
+) {
+  return moment().isAfter(moment(givenDate, givenDateFormat));
+};
+
+// from moment.js documentation: moment().isBefore() has undefined behavior and should not be used!
 export const isTodayBeforeGivenDate = function (
```

Comment:
Can you revise this function then, so that its not using isBefore? :"   I checked and there is an alternative, `isSameOrBefore`

Reply:
If not, its oke XD